handler/src/main/java/io/netty/handler/ssl/KeyManagerFactoryWrapper.java

```
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2019 The Netty Project
+ *
+ * The Netty Project licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.netty.handler.ssl;
+
+import io.netty.handler.ssl.util.SimpleKeyManagerFactory;
+import io.netty.util.internal.ObjectUtil;
+
+import java.security.KeyStore;
+import javax.net.ssl.KeyManager;
+import javax.net.ssl.ManagerFactoryParameters;
+
+final class KeyManagerFactoryWrapper extends SimpleKeyManagerFactory {
+    private final KeyManager km;
+
+    KeyManagerFactoryWrapper(KeyManager km) {
+        this.km = km;
+    }
+
+    @Override
+    protected void engineInit(KeyStore keyStore, char[] var2) throws Exception { }
+
+    @Override
+    protected void engineInit(ManagerFactoryParameters managerFactoryParameters)
+            throws Exception { }
+
+    @Override
+    protected KeyManager[] engineGetKeyManagers() {
+        ObjectUtil.checkNotNull(km, "KeyManagerFactoryWrapper is created with null KeyManager value");
```

remove this and do the check in the constructor to fail fast